### PR TITLE
Feature/build latest researchstack

### PR DESCRIPTION
**Pull request by liujoshua**
Might need to add build-tools for specific versions

### CODE REVIEW

**Comment by liujoshua** [comment]
Commit d2b6a11 should succeed, commit a048405 is expected to fail until this PR is merged https://github.com/ResearchStack/ResearchStack/pull/323

**Comment by liujoshua** [comment]
Note that this will mean this repo's `develop` builds are no longer reproducible, since `skin:2.0.0-SNAPSHOT` will change with time

File: .travis/setup.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# hack to get latest ResearchStack code since artficts aren't being published
+if [[ $TRAVIS_BRANCH != 'master' ]]; then
```
Comment (dephillipsmichael) [inline .travis/setup.sh:2]: would "$TRAVIS_BRANCH == 'develop'" make more sense?

**Comment by dephillipsmichael** [comment]
"Note that this will mean this repo's develop builds are no longer reproducible, since skin:2.0.0-SNAPSHOT will change with time" hmmm not sure if that is the behavior we want or not.  What do you think about including RS as a sub-module?  That way we can control what that sub-module points to

**Comment by liujoshua** [comment]
We could do that temporarily and change mPower's structure to mPower -> BridgeAndroidSDK -> ResearchStack

**Pull request merged**